v2/deprovision_instance.go

```diff
@@ -5,34 +5,22 @@ import (
 	"net/http"
 )
 
-// internal message body types
-
-type deprovisionInstanceRequestBody struct {
-	serviceID *string `json:"service_id"`
-	planID    *string `json:"plan_id,omitempty"`
-}
-
 func (c *client) DeprovisionInstance(r *DeprovisionRequest) (*DeprovisionResponse, error) {
 	if err := validateDeprovisionRequest(r); err != nil {
 		return nil, err
 	}
 
 	fullURL := fmt.Sprintf(serviceInstanceURLFmt, c.URL, r.InstanceID)
 
-	params := map[string]string{}
+	params := map[string]string{
+		serviceIDKey: string(r.ServiceID),
```

Comment:
Looks correct to me based on https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#parameters-4